src/Tools/ExternalAccess/OmniSharp/CodeActions/OmniSharpCodeFixContextFactory.cs

```diff
@@ -43,6 +45,20 @@ internal static class OmniSharpCodeFixContextFactory
             FixAllContext.DiagnosticProvider fixAllDiagnosticProvider,
             Func<string, OmniSharpCodeActionOptions> optionsProvider,
             CancellationToken cancellationToken)
+            => CreateFixAllContext(document, diagnosticSpan, project, codeFixProvider, scope, codeActionEquivalenceKey, diagnosticIds, DiagnosticSeverity.Hidden, fixAllDiagnosticProvider, optionsProvider, cancellationToken);
+
+        public static FixAllContext CreateFixAllContext(
+            Document? document,
+            TextSpan? diagnosticSpan,
+            Project project,
+            CodeFixProvider codeFixProvider,
+            FixAllScope scope,
+            string? codeActionEquivalenceKey,
+            IEnumerable<string> diagnosticIds,
+            DiagnosticSeverity minimumSeverity,
```

Comment:
Why is this minimum severity and not the exact severity to fix?

Reply:
Definitely needs to go through public API review. Can you please file a tracking issue and add your proposal?

Reply:
> Why is this minimum severity and not the exact severity to fix?

It seems like if you trigger the fix on a warning, it should also fix errors.

Reply:
> Definitely needs to go through public API review. Can you please file a tracking issue and add your proposal?

I'll get a tracking issue filed, but there are no API changes in this PR anymore.

Reply:
➡️ #71605